### PR TITLE
Update root620000 migration to only update root_base

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR updates the root620000 migration (#410) to only update the `root_base` package (see #414)

cc: @chrisburr @beckermr 

Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/migrations/root620000.yaml [hunk 1]
```diff
@@ -1,15 +1,11 @@
-migrator_ts: 1583150223
+migrator_ts: 1583251649
 __migrator:
   kind:
     version
   migration_number:
-    1
+    2
```
Comment (beckermr) [inline recipe/migrations/root620000.yaml]: We can keep the timestamp and migration number the same I think. 

@chrisburr for double checking this

Comment (chrisburr) [reply recipe/migrations/root620000.yaml]: Yes we can

**Comment by chrisburr** [comment]
@beckermr It's still not working, any ideas?

**Comment by beckermr** [comment]
Nope. We’ll need to dig in the graph.

**Comment by beckermr** [comment]
This might be root_base versus root-base

**Comment by chrisburr** [comment]
Does the pin have to already be rendered into the feedstock's configuration?

I've just realised the pin was added after root_numpy was last rerendered: https://github.com/conda-forge/root_numpy-feedstock/blob/master/.ci_support/linux_python3.8.yaml

**Comment by beckermr** [comment]
The package needs to be in the metadata of the graph. So if the bot is looking for `root_base` but it is denoted as `root-base`, then the bot won't find any feedstocks to migrate. This metadata is parsed out of the meta.yaml files, so my guess is that this is the issue. 

Can you bump us in the bot gitter room? Others can confirm there.

**Comment by chrisburr** [comment]
Where is `root-base` used? It doesn't exist on Anaconda.org and in recipes I can only find instances of `root_base`?

**Comment by beckermr** [comment]
Ohhhhhh. Well then this is a bug.

**Comment by chrisburr** [comment]
I've tried asking the bot to rerender in an issue but it hasn't responded: https://github.com/conda-forge/root_numpy-feedstock/issues/8

Doing it by hand it works so it's not an conda-smithy issue. As I don't think I have access, can you check the logs in case that explains it?

**Comment by beckermr** [comment]
issue rerenders are broken sorry.

**Comment by beckermr** [comment]
https://github.com/conda-forge/conda-forge-webservices/issues/326

**Pull request merged**